llvm/test/tools/yaml2obj/GOFF/GOFF-basic.yaml

```diff
@@ -0,0 +1,59 @@
+# RUN: yaml2obj %s | od -v -An -tx1 | FileCheck --ignore-case %s
+
+--- !GOFF
+FileHeader:
+  ArchitectureLevel: 1
+Records:
+# Bytes 80-159
+# CHECK: 03 00 00 00 00 00 00 01 00 00 00 00 00 00 00 00
+# CHECK: 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00
+# CHECK: 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00
+# CHECK: 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00
+# CHECK: 00 00 00 00 00 00 00 04 88 89 7b c3 00 00 00 00
+  - Symbol:
+      Name:              'hi#C'
+      Type:              ESD_ST_SectionDefinition
+      ID:                1
+      OwnerID:           0
+      Address:           0
+      Length:            0
+      NameSpace:         ESD_NS_ProgramManagementBinder
+# Bytes 160-239
+# CHECK: 03 00 00 01 00 00 00 02 00 00 00 01 00 00 00 00
+# CHECK: 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00
+# CHECK: 00 00 00 00 00 00 00 00 03 81 00 00 00 00 00 00
+# CHECK: 00 00 00 00 00 00 00 00 00 00 00 00 04 04 01 01
+# CHECK: 00 40 03 00 00 00 00 07 c3 6d e6 e2 c1 f6 f4 00
+  - Symbol:
+      Name:              'C_WSA64'
+      Type:              ESD_ST_ElementDefinition
+      ID:                2
+      OwnerID:           1
+      NameSpace:         ESD_NS_Parts
+      Flags:             [ ESD_FillByteValuePresent, ESD_RQ_1 ]
+      FillByteValue:     0
+      Amode:             ESD_AMODE_64
+      Rmode:             ESD_RMODE_64
+      BindingAlgorithm:  ESD_BA_Merge
+      Executable:        ESD_EXE_DATA
+      LoadingBehavior:   ESD_LB_Deferred
+      Alignment:         ESD_ALIGN_Doubleword
+# Bytes 240-319
+# CHECK: 03 00 00 03 00 00 00 03 00 00 00 02 00 00 00 00
+# CHECK: 00 00 00 00 00 00 00 00 00 00 00 02 00 00 00 00
+# CHECK: 00 00 00 00 00 00 00 00 03 00 00 00 00 00 00 00
+# CHECK: 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 01
+# CHECK: 00 01 24 00 00 00 00 04 88 89 7b e2 00 00 00 00
+  - Symbol:
+      Name:              'hi#S'
+      Type:              ESD_ST_PartReference
+      ID:                3
+      OwnerID:           2
+      Address:           0
+      Length:            2
+      NameSpace:         ESD_NS_Parts
+      LinkageType:       ESD_LT_XPLink
+      Alignment:         ESD_ALIGN_Quadword
+      BindingScope:      ESD_BSC_Section
+      Executable:        ESD_EXE_DATA
+
```

Comment:
Nit: Looks like you've got an extra blank line at EOF. Please delete it (files should end in precisely one `\n`).